providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/GrpcConnector.java

```diff
@@ -104,7 +105,7 @@ public void shutdown() throws Exception {
                 this.channel.awaitTermination(this.deadline, TimeUnit.MILLISECONDS);
                 log.warn(String.format("Unable to shut down channel by %d deadline", this.deadline));
             }
-            this.stateConsumer.accept(false, Collections.emptyList());
+            this.onConnectionEvent.accept(new ConnectionEvent(false, Collections.emptyList(), Collections.emptyMap()));
```

Comment:
Do we need the empty map and list here? Can we maybe add this also as defaults to the connectionEvent?

Reply:
Oh we do not. I actually have other constructors, I'm just not using them here. Will update.

Reply:
Done, I think this was the only case.